package.json

```diff
@@ -4,7 +4,9 @@
     "build": "turbo run build",
     "dev": "turbo run dev",
     "lint": "turbo run lint",
+    "lint:strict": "eslint --max-warnings=0 apps/**/*.{ts,tsx}",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
+    "format:check": "prettier -c \"**/*.{ts,tsx,md}\"",
```

Comment:
Same for prettier we can run it in every package and run it with turbo ?

Reply:
For prettier, I am currently not sure we can run that with turbo, I went through several articles and what they had implemented was equivalent to what I wrote.

Reply:
@thibaultleouay please provide me some updates so I proceed. Thanks